dhall-nixpkgs/dhall-nixpkgs.cabal

```diff
@@ -19,7 +19,7 @@ Executable dhall-to-nixpkgs
   Build-Depends:       base                 >= 4.11     && < 5
                      , aeson                >= 1.0.0.0  && < 2.1
                      , base16-bytestring    >= 1.0.0.0
-                     , base64-bytestring    >= 1.2.1.0
+                     , base64-bytestring    >= 1.1.0.0
```

Comment:
I think I picked that bound based on this changelog entry:

> 1.2.1.0
> Bugfix for GHC 9.0.1 memory corruption bug (#46)
> Thanks to Fraser Tweedale and Andrew Lelechenko for logging and helping with this fix.

I'm not familiar with the details though.

If it's important to have a lower bound with GHC 8.10, I'd suggest that we keep the current bound for GHC >= 9:

```
if impl(ghc >= 9)
  Build-Depends: base64-bytestring >= 1.2.1.0
```

Reply:
My view is that the `dhall` package is not the place to enforce that decision and if we build against an older version then we should permit it in the bounds.  If we really wanted to prevent the `base64-bytestring-1.1.0.0` package from being used on `ghc-9.0` then I think the right place to enforce that is in the `base64-bytestring` package.  Specifically, they could add a revision to `base64-bytestring.cabal` for version `1.1.0.0` to set `Buildable: false` for that version of `ghc`

Reply:
Ok.